File: binarysearch/bubblesortlist.js
const fs = require("fs");

const dataobj = JSON.parse(fs.readFileSync("data1.json", "utf8"));

const data = dataobj.data;

for (let i = 0; i < data.length; i++) {
  for (let j = 0; j < data.length; j++) {
    if (data[i] < data[j]) {
      let temp = data[i];
      data[i] = data[j];
      data[j] = temp;
    }
  }
}
console.log(data);

fs.writeFile("data1.json", JSON.stringify(data), function err() {
  console.log("ayo");
});

File: binarysearch/bubblesortlist.py
import json

with open("data1.json", "r") as f:
    data = json.load(f)

for x in data["data"]:
    s = True
    for y in range(len(data["data"]) - 1):
        if data["data"][y] > data["data"][y+1]:
            temp = data["data"][y]
            data["data"][y] = data["data"][y+1]
            data["data"][y+1] = temp
            s = False
    if s:
        break

with open("data1.json", "w") as f:
    json.dump(data, f)
